FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/TCPEchoClient_SingleTasks.c

```diff
@@ -399,9 +399,9 @@
             eDHCPCallbackAnswer_t xApplicationDHCPHook( eDHCPCallbackPhase_t eDHCPPhase,
```

Comment:
The same unused attribute is needed here too, right? Can you suppress the warnings like the following for consistency with other code:

```c
    #if ( ipconfigUSE_DHCP_HOOK != 0 )

        #if ( ipconfigIPv4_BACKWARD_COMPATIBLE == 1 )
            eDHCPCallbackAnswer_t xApplicationDHCPHook( eDHCPCallbackPhase_t eDHCPPhase,
                                                        uint32_t ulIPAddress )
            {
                ( void ) eDHCPPhase;
                ( void ) ulIPAddress;

                return eDHCPContinue;
            }
        #else /* ( ipconfigIPv4_BACKWARD_COMPATIBLE == 1 ) */
            eDHCPCallbackAnswer_t xApplicationDHCPHook_Multi( eDHCPCallbackPhase_t eDHCPPhase,
                                                              struct xNetworkEndPoint * pxEndPoint,
                                                              IP_Address_t * pxIPAddress )
            {
                ( void ) eDHCPPhase;
                ( void ) pxEndPoint;
                ( void ) pxIPAddress;

                return eDHCPContinue;
            }
        #endif /* ( ipconfigIPv4_BACKWARD_COMPATIBLE == 1 ) */

    #endif /* if ( ipconfigUSE_DHCP_HOOK != 0 )*/
```

Reply:
Right. I've changed all these occurences to your proposal.